app/controllers/shopping_list_controller.rb

```diff
@@ -0,0 +1,5 @@
+class ShoppingListController < ApplicationController
+  def index
+    @recipe_foods = current_user.recipe_foods.includes(:food)
+  end
+end
```

Comment:
- According to the project requirements, `the general shopping list should count the total food items and total price of the missing food`. As can be seen in your project, the values for the count and sum are 0, even if there are items in the shopping list. Please fix this 🙏 
Your project:
![tethet](https://user-images.githubusercontent.com/73607512/236638006-b7df88b1-4a2e-492f-89c2-ab0b3384609c.PNG)